pkgs/tools/text/xidel/default.nix

```diff
@@ -72,20 +80,11 @@ in stdenv.mkDerivation rec {
     cp xidel "$out/bin/"
   '';
 
-  doCheck = true;
-
+  # disabled, because tests require network
   checkPhase = ''
-    # Not all, if any, of these tests are blockers. Failing or not this phase will pass.
-    # As of 2021-08-15, all of 37 failed tests are linked with the lack of network access.
     ./tests/tests.sh
   '';
 
-  doInstallCheck = true;
-
-  installCheckPhase = ''
-    $out/bin/xidel --version | grep "${version}"
```

Comment:
Why are you removing the version check?

Reply:
because this isn't a released version

Reply:
Then we should update `version = ...`, even if disabling the test.

Reply:
done. anything else to do here?